playbooks/vendor-evaluations.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Vendor Evaluations
+description: How to evaluate 3rd party vendors
+---
+
+Evaluting vendors can be daunting and overwhelming. Here are list of considerations when evaluation a vendor,
+grouped by larger themes:
+
+**Cost**
+- <details><summary>ROI</summary>
```

Comment:
It would be worth also adding a bullet-point here just for "What is the cost of the vendor? How does their fee structure scale?"